test/forge/BaseTest.sol

```diff
@@ -189,7 +189,7 @@ contract BaseTest is Test {
             amountBorrowed.wDivDown(marketParams.lltv).mulDivDown(ORACLE_PRICE_SCALE, priceCollateral);
         amountCollateral = bound(amountCollateral, 0, Math.min(maxCollateral, MAX_COLLATERAL_ASSETS));
 
-        vm.assume(amountCollateral > 0);
+        vm.assume(amountCollateral > 0 && amountCollateral < maxCollateral);
```

Comment:
is `bound` broken? I don't get how we could have amountCollateral>=maxCollateral

Reply:
re: no it's not, but we should exclude maxCollateral because otherwise the account is healthy. And bounding to maxCollateral-1 breaks other places, because it's sometimes 0.